src/main/lens-api.ts

```diff
@@ -1,17 +1,15 @@
 import http from "http";
```

Comment:
guess this file could be now named `src/main/utils/http-respond.ts` or similar?

Reply:
Probably would be a better name.